typings/@elastic/eui/index.d.ts

```diff
@@ -19,7 +19,3 @@
 import { Direction } from '@elastic/eui/src/services/sort/sort_direction';
 
 // TODO: Remove once typescript definitions are in EUI
```

Comment:
Is this file removable now?

Reply:
Looks like! I'm planning on following up with deleting four files, which are the remaining EUI type overrides.

typings/@elastic/eui/index.d.ts
typings/@elastic/eui/lib/format.d.ts
typings/@elastic/eui/lib/services.d.ts
x-pack/typings/@elastic/eui/index.d.ts

Which will need very small corrections to 2 other files